src/Services/QueuedJobService.php

```diff
@@ -66,6 +66,37 @@ class QueuedJobService
      */
     private static $stall_threshold = 3;
 
+    /**
+     * How early stuck jobs will become eligible for automated retry processing (minutes)
+     *
+     * @config
+     * @var int
+     */
+    private static int $job_retry_sentinel = 1;
```

Comment:
Can you please explain the purpose of this? The documentation says "represents separation of job retries and job processing mechanism to avoid potential edge cases" but it's not clear what sort of edge cases it's avoiding or why this separation is needed at a global level.

Reply:
The naming is probably a bit unfriendly as well - the word "sentinel" is pretty esoteric. Maybe something like `job_retry_buffer` as in it's the buffer between when the job first ran and its first retry (if that's what it is)?